certz/certz.proto

```diff
@@ -636,8 +641,108 @@ message CertificateSigningRequest {
 // the specified parameters.
 message GenerateCSRRequest {
   CSRParams params = 1;
+
+  // Indicates measurements that are prerequisites for certificate
+  // provisioning.
+  repeated ReferenceIntegritySpec integrity = 2;
 }
+
 // A message sent by the target as response to the GenerateCSRRequest.
 message GenerateCSRResponse {
   CertificateSigningRequest certificate_signing_request = 1;
+
+  // Reference integrity measurements as specified in the request.
+  repeated ReferenceIntegrityData integrity = 2;
+}
+
+message GetIntegrityManifestRequest {}
+
+message GetIntegrityManifestResponse {
+  // The manifest is a generic byte stream for passing protocol dependent
+  // (unsecured) data required prior to returning attestation data.
```

Comment:
can we add enough a comment to say this will migrated to structured data that will be a reference once defined

Reply:
Done.